setup.py

```diff
@@ -1,8 +1,62 @@
-from distutils.core import setup
-from Cython.Build import cythonize
-import numpy
-
-setup(
-    ext_modules=cythonize("*.pyx", language="c++"),
-    include_dirs=[numpy.get_include()]
-)
+"""
+Implements the MDLP discretization criterion from Usama Fayyad's paper
```

Comment:
You might be interested in this: http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html#example-numpy

Reply:
I would also recommend making `setup.py` an executable file.

Reply:
`chmod +x setup.py` :)